README.md

```diff
@@ -106,6 +107,14 @@ Setup on kubernetes is complex and requires prior knowledge about linux systems,
 
 ## Customization
 
+### Custom Django Settings
+
+If you want to add custom Django settings, you can follow the following guide.
+
+1. Create a directory `customize` in the same location as the `docker-compose.yml` file.
+2. Create a sub-directory `customize/configuration`.
+3. Create a file which will contain settings for Django `customize/configuration/custom_django_settings.py`. You can write all Django specific configuration in this file.
```

Comment:
we can allow to add a directory instead of only one file and hence allow to add multiple python files to be loaded (which can be used to create custom apps or implement custom signal receivers).

Reply:
I think we need an instruction like:

```
cp <path>/custom_django_settings.example.py <path>/custom_django_settings.py
```

Eg:

```
cp customization/django/settings.example.py customization/django/settings.py
```